test/functional/notes_controller_test.rb

```diff
@@ -409,7 +409,7 @@ def teardown
 
     assert_template 'editor/post'
     selector = css_select '.alert'
-    assert_equal selector.size, 2
+    assert_equal selector.size, 1
```

Comment:
Hey @Manasa2850 , this should not change, right? 
We have just used another partial `errorMessages` instead of old code and both are using `alert ` class.

Reply:
Hi @Sagarpreet. Yes you're right...I had made that change while testing it locally since there were some errors. I'll change it back...sorry about that....I guess I'll need to make changes to the controller to fix those errors
Thank you!